build.gradle

```diff
@@ -24,23 +24,34 @@ subprojects {
         addons
     }
 
-    // rosuda jars are required for both par-connector and par-script
-    dependencies {
-        addons "org.rosuda:jriengine:0.5.x"
-        addons "org.rosuda:jri:0.5.x"
-        addons "org.rosuda:rengine:1.7.x"
-    }
 }
 
-project(':par-script') {
+def isWindows = System.properties['os.name'].toLowerCase().contains('windows')
+def isMac = System.properties['os.name'].toLowerCase().contains('mac')
```

Comment:
these variables are set at the gradle build toplevel, where no dependencies have been declared. How can I use here OperatingSystem.current().isMacOsX() ?